core/templates/domain/objects/objects-domain.constants.ts

```diff
@@ -68,4 +75,304 @@ export const ObjectsDomainConstants = {
     INCLUDES_ZERO: 'I18N_INTERACTIONS_RATIO_INCLUDES_ZERO',
     EMPTY_STRING: 'I18N_INTERACTIONS_RATIO_EMPTY_STRING',
   },
+
+  // Used for converting units to their canonical forms.
```

Comment:
Where is the test that confirms this is a complete list (i.e. if you can enter something in mathjs then it would be found here)?

If this isn't a complete list then what happens when the user enters something that is not in mathjs?